### PR TITLE
refactor: http files moved to one file within examples folder

### LINKED ISSUE #37: Put all http example snippet in one file

### PR DESCRIPTION
**Pull request by marktlinn**
Http files moved to one file

### CODE REVIEW

File: src/examples/index.ts [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-import httpGETWithURLParams from "./http-get-with-query-string";
-import httpPOSTWithAuthBearer from "./http-post-with-auth-bearer";
-import httpPOSTWithFormSubmit from "./http-post-form-submit";
-import httpPOSTWithJSONBody from "./http-post-with-json-body";
-import httpPUTWithFileUpload from "./http-put-with-file-upload";
+import { httpGETWithURLParams } from "./examples";
+import { httpPOSTWithAuthBearer } from "./examples";
+import { httpPOSTWithFormSubmit } from "./examples";
+import { httpPOSTWithJSONBody } from "./examples";
+import { httpPUTWithFileUpload } from "./examples";
```
Comment (0hsn) [inline src/examples/index.ts]:
```suggestion
import {
  httpGETWithURLParams,
  httpPOSTWithAuthBearer,
  httpPOSTWithFormSubmit,
  httpPOSTWithJSONBody,
  httpPUTWithFileUpload
} from "./examples";
```

**Review by 0hsn — CHANGES_REQUESTED** [review]
rename `src/examples/examples.ts` to `src/snippets/http.ts`

**Comment by arifszn** [comment]
@marksist300, first off, thanks for taking the time to contribute! 🎉👍

Great PR. The only thing I have a concern about is if the snippets grow more in the future, or if one snippet is too large, then in one file will it look crowded?

**Comment by marktlinn** [comment]
> rename `src/examples/examples.ts` to `src/snippets/http.ts`

Done 👍🏻

**Comment by 0hsn** [comment]
@arifszn 
> Great PR. The only thing I have a concern about is if the snippets grow more in the future, or if one snippet is too large, then in one file will it look crowded?

I don't see more than 2 snippet getting added to the file, in coming future, or no-more snippet gets added at all. So, it's okayish. But, good point, thanks for raising it 🙏

**Review by 0hsn — CHANGES_REQUESTED** [review]
@marksist300 Thanks for the work! really happy to see you around.

One last review would be: can you also move code from `src/snippets/minimal-request.ts` to `src/snippets/http.ts`, then import and use it?

**Comment by marktlinn** [comment]
> 



> @marksist300 Thanks for the work! really happy to see you around.
> 
> One last review would be: can you also move code from `src/snippets/minimal-request.ts` to `src/snippets/http.ts`, then import and use it?

Done. Let me know if there are any other changes required.

**Review by arifszn — APPROVED** [review]
Ok from me.

File: src/snippets/index.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import {
+  httpGETWithURLParams,
+  httpPOSTWithAuthBearer,
+  httpPOSTWithFormSubmit,
+  httpPOSTWithJSONBody,
+  httpPUTWithFileUpload,
+  minimalRequest,
+} from "./http";
+
+console.log(httpGETWithURLParams);
```
Comment (arifszn) [inline src/snippets/index.ts]: Remove this please.

Comment (marktlinn) [reply src/snippets/index.ts]: Of course, thank you for letting me know. Done 😊

**Review by 0hsn — APPROVED** [review]
👏

**Pull request merged**